codespell_lib/data/dictionary.txt

```diff
@@ -2340,6 +2340,7 @@ hospitible->hospitable
 hould->hold, should,
 hounour->honour
 housr->hours, house,
+hove->have, hover, love,
```

Comment:
turns out `hove` is the past tense of `heave`, which I didn't know until I checked